simplyblock_core/test/test_parsing_thread_siblings.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from simplyblock_core.utils import parse_thread_siblings_list
+
+
+class TestParseThreadSiblingsList(unittest.TestCase):
```

Comment:
I'd like these tests to use the pytest parametrize pattern used in other tests.

Reply:
What does that mean? 😅